src/lib/lightning/lnd/lndService.ts

```diff
@@ -45,13 +45,27 @@ class LndService implements LightningService {
     } = await proxy.pendingChannels(this.cast(node));
 
     const pluckChan = (c: any) => c.channel as LND.PendingChannel;
+    const isChanInitiatorLocal = (c: LND.PendingChannel | LND.Channel) =>
+      c.initiator === LND.Initiator.INITIATOR_LOCAL;
     // merge all of the channel types into one array
     return [
-      ...open.filter(c => c.initiator).map(mapOpenChannel),
-      ...opening.map(pluckChan).map(mapPendingChannel('Opening')),
-      ...closing.map(pluckChan).map(mapPendingChannel('Closing')),
-      ...forceClosing.map(pluckChan).map(mapPendingChannel('Force Closing')),
-      ...waitingClose.map(pluckChan).map(mapPendingChannel('Waiting to Close')),
+      ...open.filter(isChanInitiatorLocal).map(mapOpenChannel),
```

Comment:
THis won't work because `LND.Channel.initiator` is a `boolean`, but the `isChanInitiatorLocal` function is comparing it to an `enum`/`number`. That function looks good for all of the pending arrays below, but the open channels array has a different type so it should be filtered differently.

Reply:
Ah ty!